src/TreeMenu/index.tsx

```diff
@@ -27,7 +27,8 @@ export type TreeMenuProps = {
   locale?: LocaleFunction;
   matchSearch?: MatchSearchFunction;
   disableKeyboard?: boolean;
-};
+  onKeyDown?: () => any;
```

Comment:
The typing is inconsistent with the others. It should be `onKeyDown?: (event: KeyboardEvent<HTMLDivElement>) => void`?